helm/prometheus-operator-app/templates/prometheus-operator/crds.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (eq .Release.Service "Tiller") .Values.prometheusOperator.enabled .Values.prometheusOperator.createCustomResource -}}
+{{- if and .Values.prometheusOperator.enabled .Values.prometheusOperator.createCustomResource -}}
```

Comment:
I think we need to keep release.service eq Tiller here to avoid CRD issues between helm 2/3

Reply:
They mention crd issues in the [upstream chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#prerequisites) but it seems it's only an issue with <2.14 
@rossf7 can you confirm, that we need the above workaround?

Reply:
that would need to stay. it causes different behaviour based on whether tiller is being used to deploy:

helm 2:
- it will create a single manifest from all the files in `/crds`

helm 3:
- this manifest will be blank and helm will install crds from `/crds` using the new deployment method.